pandas/tests/io/test_parquet.py

```diff
@@ -1005,6 +992,15 @@ def test_timezone_aware_index(self, request, pa, timezone_aware_date_list):
         expected = df[:]
         if pa_version_under11p0:
             expected.index = expected.index.as_unit("ns")
+        if timezone_aware_date_list.tzinfo != datetime.timezone.utc:
+            # pyarrow returns pytz.FixedOffset while pandas constructs datetime.timezone
+            # https://github.com/pandas-dev/pandas/issues/37286
+            import pytz
```

Comment:
```suggestion
            pytz = pytest.importorskip("pytz")
```
I suppose your prior import was safe since having pyarrow installed would require pytz to also be installed here? IMO would be nice to extra safe with `importorskip` in case that ever changes :)

Reply:
I actually first wrote it like that, but then intentionally did a plain import. Because otherwise, if we would for some reason no longer have pytz in the test builds that have pyarrow, we would silently never run this test. I would rather be notified at that point that we need pytz.

Now, while writing that, if the env doesn't have pytz, then pyarrow would actually fall back to another tz implementation. We could also write the test more generically so it does not only work with pytz. But I think at the moment we install pytz in almost all test envs?

Reply:
> But I think at the moment we install pytz in almost all test envs?

Yeah tests envs where we install optional dependencies will have pyarrow and pytz. Sure I'm OK then skipping the `importorskip`

Reply:
Actually looks like we need to install `pytz` for the pyarrow nightly build i.e. add pyarrow to `actions-311-pyarrownightly.yaml` https://github.com/pandas-dev/pandas/actions/runs/11595223185/job/32283356899?pr=60143

(I guess my above comment didn't really apply to our one-off builds like the pyarrow nightly)

Reply:
OK, made the test robust with both with and without pytz; seems fine to test both cases